example/android/app/build.gradle

```diff
@@ -47,8 +47,8 @@ android {
         applicationId "de.enough.example"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-build-configuration.
-        minSdkVersion 21
-        targetSdkVersion flutter.targetSdkVersion
+        minSdkVersion 19
+        targetSdkVersion 34
```

Comment:
if you target it, maybe need to change too much in Tmail, please use 33

Reply:
It's just an example run test with no impact